tracer/src/Datadog.InstrumentedAssemblyGenerator/AssemblyGeneratorArgs.cs

```diff
@@ -7,21 +7,25 @@ namespace Datadog.InstrumentedAssemblyGenerator
 {
     public class AssemblyGeneratorArgs
     {
-        public AssemblyGeneratorArgs(string instrumentationLogsBaseFolder, string[] modulesToVerify = null)
+        public AssemblyGeneratorArgs(string instrumentationLogsBaseFolder, bool copyOriginalModulesToDisk = false, string[] modulesToVerify = null)
         {
             InstrumentationLogsBaseFolder = instrumentationLogsBaseFolder;
             ModulesToGenerate = modulesToVerify ?? Array.Empty<string>();
+            if (copyOriginalModulesToDisk)
+            {
+                OriginalModulesFolder = Path.Combine(InstrumentationLogsBaseFolder, InstrumentedAssembliesFolderName);
```

Comment:
```suggestion
                // If DD_COPY_ORIGINALS_MODULES_TO_DISK was enabled, this folder will contain all the assemblies that were loaded at runtime, and we'll read the original assemblies from there exclusively.
                // Otherwise, we assume are running in the same machine as the customer application and that files haven't changed, and load the assemblies from their original locations.
                
                OriginalModulesFolder = Path.Combine(InstrumentationLogsBaseFolder, InstrumentedAssembliesFolderName);
```

Reply:
Fixed in d6e34e581c70cc2271806c12db2c3d4f2caf78f1